src/main/java/com/gmail/goosius/siegewar/playeractions/InvadeTown.java

```diff
@@ -105,10 +105,10 @@ private static void allowInvasionOrThrow(Player player, Nation residentsNation,
 			throw new TownyException(translator.of("msg_err_action_disable"));
 
 		if (!TownyUniverse.getInstance().getPermissionSource().testPermission(player, SiegeWarPermissionNodes.SIEGEWAR_NATION_SIEGE_INVADE.getNode()))
-			throw new TownyException(translator.of("msg_err_action_disable"));
+			throw new TownyException(translator.of("msg_err_cannot_invade_not_enough_permissions"));
 
 		if(residentsNation == null)
-			throw new TownyException(translator.of("msg_err_action_disable"));  //Can't invade if nationless
+			throw new TownyException(translator.of("msg_err_cannot_invade_not_enough_permissions"));  //Can't invade if nationless
```

Comment:
I ran into this issue somewhere else and it was very bothersome, with admins thinking they had not set up their permissions correctly.
```suggestion
			throw new TownyException(translator.of(""msg_err_dont_belong_nation""));  //Can't invade if nationless
```